### ISSUE TITLE
jupyterlab_pyviz does not support jupyterlab 3.0.0

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#1873]
jupyterlab: 3.0.0
panel: 10.2.0
(Both installed in a virtual environment using pip).

When I run `jupyter labextension install @pyviz/jupyterlab_pyviz` I get

```bash
ValueError: The extension "@pyviz/jupyterlab_pyviz" does not yet support the current version of JupyterLab.
```

The full log is

```bash
Node v12.16.3

Yarn configuration loaded.
> "C:\Program Files\nodejs\npm.CMD" pack @pyviz/jupyterlab_pyviz
npm notice 
npm notice package: @pyviz/jupyterlab_pyviz@1.0.4
npm notice === Tarball Contents === 
npm notice 1.7kB   package.json     
npm notice 1.5kB   LICENSE.txt      
npm notice 2.2kB   README.md        
npm notice 916.2kB dist/index.js    
npm notice 3.7MB   dist/index.js.map
npm notice 64B     lib/index.d.ts   
npm notice 153B    lib/index.js     
npm notice 622B    lib/manager.d.ts 
npm notice 1.3kB   lib/manager.js   
npm notice 679B    lib/plugin.d.ts  
npm notice 2.1kB   lib/plugin.js    
npm notice 2.1kB   lib/renderer.d.ts
npm notice 8.8kB   lib/renderer.js  
npm notice 419B    lib/widgets.d.ts 
npm notice 5.6kB   lib/widgets.js   
npm notice === Tarball Details === 
npm notice name:          @pyviz/jupyterlab_pyviz                 
npm notice version:       1.0.4                                   
npm notice filename:      pyviz-jupyterlab_pyviz-1.0.4.tgz        
npm notice package size:  1.1 MB                                  
npm notice unpacked size: 4.7 MB                                  
npm notice shasum:        f2ef3b2d1fcbd7904fd2eac06584c2747d9db01c
npm notice integrity:     sha512-fLO+9tJd9RGZW[...]I2+V0HV5mgBUg==
npm notice total files:   15                                      
npm notice 
pyviz-jupyterlab_pyviz-1.0.4.tgz

Fetching URL: https://registry.yarnpkg.com/@pyviz%2Fjupyterlab_pyviz
Fetching URL: https://registry.yarnpkg.com/@pyviz%2Fjupyterlab_pyviz
Traceback (most recent call last):

  File "c:\repos\private\awesome-panel-starter\.venv\lib\site-packages\jupyterlab\debuglog.py", line 47, in debug_logging
    yield

  File "c:\repos\private\awesome-panel-starter\.venv\lib\site-packages\jupyterlab\labextensions.py", line 127, in start
    ans = self.run_task()

  File "c:\repos\private\awesome-panel-starter\.venv\lib\site-packages\jupyterlab\labextensions.py", line 178, in run_task
    for i, arg in enumerate(self.extra_args)

  File "c:\repos\private\awesome-panel-starter\.venv\lib\site-packages\jupyterlab\labextensions.py", line 178, in <listcomp>
    for i, arg in enumerate(self.extra_args)

  File "c:\repos\private\awesome-panel-starter\.venv\lib\site-packages\jupyterlab\commands.py", line 392, in install_extension
    return handler.install_extension(extension, pin=pin)

  File "c:\repos\private\awesome-panel-starter\.venv\lib\site-packages\jupyterlab\commands.py", line 614, in install_extension
    info = self._install_extension(extension, tempdir, pin=pin)

  File "c:\repos\private\awesome-panel-starter\.venv\lib\site-packages\jupyterlab\commands.py", line 1677, in _install_extension
    raise ValueError(msg)

ValueError: The extension "@pyviz/jupyterlab_pyviz" does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab                        Extension      Package
>=3.0.0 <3.1.0                    >=2.0.0 <3.0.0 @jupyterlab/application
>=3.0.0 <3.1.0                    >=2.0.0 <3.0.0 @jupyterlab/apputils
>=3.0.0 <3.1.0                    >=2.0.0 <3.0.0 @jupyterlab/notebook
>=3.0.0 <3.1.0                    >=2.0.0 <3.0.0 @jupyterlab/rendermime-interfaces
>=6.0.0 <6.1.0                    >=5.0.0 <6.0.0 @jupyterlab/services

Exiting application: lab

```

**Comment by philippjfr** [comment]
Started here: https://github.com/holoviz/holoviz_jlab

Not 100% if that's the best approach but creating a new package certainly seems easiest.

**Comment by philippjfr** [comment]
Just released `pyviz_comms` 2.0 which automatically bundles the JupyterLab extension. Simply upgrade with `pip install pyviz_comms --upgrade` or `conda install -c pyviz pyviz_comms`.
